package.json

```diff
@@ -413,7 +413,6 @@
         "watch": "npm run watch-wp",
         "watch-wp": "webpack --watch",
         "lint": "tslint -c tslint.json -p tsconfig.json",
-        "postinstall": "node ./node_modules/vscode/bin/install",

```

Comment:
was this remove intentional?

Reply:
Yes see: https://code.visualstudio.com/api/working-with-extensions/testing-extension#migrating-from-vscode

I'll add this link to the main PR desc. Thanks!